src/components/details/details.ts

```diff
@@ -113,7 +113,7 @@ export default class SlDetails extends LitElement {
       // Show
       emit(this, 'sl-show');
 
-      await stopAnimations(this);
+      await stopAnimations(this.body);
```

Comment:
This will stop all active animations on the page, but we only want to stop the ones causing the bug.

Reply:
Never mind. I read `this.body` as `document.body`. This is a good fix.

Cheers!

Reply:
As far as I can tell `this.body` refers to the element with class `.details__body`

Reply:
Yep, false alarm. Cheers for another solid PR!

Reply:
Thanks for getting them through so quickly :)